_includes/c11-cpp11-and-beyond-and-toolchains.md

```diff
@@ -204,12 +221,34 @@ matrix:
       
 You can find the `clang` version shipped by Xcode [here](https://trac.macports.org/wiki/XcodeVersionInfo).
 
+### Clang on FreeBSD
+
+> Clang is the default compiler on FreeBSD
+
+FreeBSD ships with Clang 8.0.1
+
+To upgrade Clang to a more recent version, install the appropriate version from packages; see below for examples:
+
+```yaml
+os: freebsd
+addons:
+    pkg:
+      - llvm90
+    env:
+      - CC=clang
+      - CXX=clang++
```

Comment:
`llvm90` installs these tools as `/usr/local/bin/clang90` and `/usr/local/bin/clang++90`.